security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -73,3 +73,63 @@ Using the `unsupportedConfigOverrides` section to modify the configuration of an
+==== Authenticating the {cert-manager-operator} with AWS Security Token Service
+* You can now configure cloud credentials for the {cert-manager-operator} on the AWS Security Token Service (STS) cluster and the AWS cluster without STS.
+
+[id="cert-manager-authentication-gcp-1.11.1"]
+==== Authenticating the {cert-manager-operator} with GCP Workload Identity
+* You can now configure cloud credentials for the {cert-manager-operator} on the GCP Workload Identity cluster and the GCP cluster without Workload Identity.
+
+[id="cert-manager-customizing-api-fields-1.11.1"]
+==== Customizing {cert-manager-operator} API fields
+* You can customize the {cert-manager-operator} API fields by overriding environment variables and arguments.
+
+[id="cert-manager-monitoring-1.11.1"]
+==== Enabling monitoring for the {cert-manager-operator}
+* You can enable monitoring and metrics collection for the {cert-manager-operator} by using a service monitor to perform the custom metrics scraping. After you have enabled monitoring for the {cert-manager-operator}, you can query its metrics by using the {product-title} web console.
+
+[id="cert-manager-dns01-acme-aws-1.11.1"]
+==== Configuring an ACME issuer to solve DNS01 challenges by using ambient credentials on AWS
+* You can use the {cert-manager-operator} to set up an ACME issuer to solve DNS-01 challenges by using ambient credentials on AWS.
+
+[id="cert-manager-dns01-acme-gcp-1.11.1"]
+==== Configuring an ACME issuer to solve DNS01 challenges on GCP
+* You can use the {cert-manager-operator} to set up an ACME issuer to solve DNS-01 challenges by using ambient and explicit credentials on GCP.
+
+[id="cert-manager-dns01-acme-azure-1.11.1"]
+==== Configuring an ACME issuer to solve DNS01 challenges on Microsoft Azure
+* You can use {cert-manager-operator} to set up an ACME issuer to solve DNS-01 challenges by using explicit credentials on Microsoft Azure.
+
+[id="cert-manager-operator-1.11.1-bug-fixes"]
+=== Bug fixes
+
+* Previously, the `cm-acme-http-solver` pod did not use the latest published Red Hat image `registry.redhat.io/cert-manager/jetstack-cert-manager-acmesolver-rhel9`. With this release, the `cm-acme-http-solver` pod uses the latest published Red Hat image `registry.redhat.io/cert-manager/jetstack-cert-manager-acmesolver-rhel9`. (link:https://issues.redhat.com/browse/OCPBUGS-10821[*OCPBUGS-10821*])
+
+* Previously, the {cert-manager-operator} did not support changing labels for cert-manager pods such as controller, CA injector, and Webhook pods. With this release, you can add labels to cert-manager pods. (link:https://issues.redhat.com/browse/OCPBUGS-8466[*OCPBUGS-8466*])
+
+* Previously, the `operatorLogLevel` field did not update the log verbosity level in the {cert-manager-operator}. Now, the `operatorLogLevel` field properly updates the log verbosity level in the {cert-manager-operator}. (link:https://issues.redhat.com/browse/OCPBUGS-9994[*OCPBUGS-9994*])
+
```

Comment:
More bug fixes we needa add to this list:
1. OCPBUGS-9960
2. OCPBUGS-9998
3. OCPBUGS-14129
4. OCPBUGS-9954